src/features/components/templates/banners/blade-shape-banner/types.ts

```diff
@@ -18,7 +18,7 @@ export type ExtraInfoType = {
 export type QRScanBoxType = {
     icon: string
     heading_one: TString
-    heading_two: TString
+    heading_two: string
```

Comment:
Previous one ok. It will be fixed from translation end.

Reply:
unfortunately no, because of new file